apis/cc-infix-en_GB/atrium-api-cc-infix-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/infix/en_GB/CharSequenceAssertionsSpec.kt

```diff
@@ -30,38 +34,41 @@ class CharSequenceAssertionsSpec : ch.tutteli.atrium.spec.integration.CharSequen
 
         private fun containsDefaultTranslationOf(plant: Assert<CharSequence>, expected: Translatable, otherExpected: Array<out Translatable>): Assert<CharSequence> {
             return if (otherExpected.isEmpty()) {
-                plant contains expected.getDefault()
+                plant.asExpect().contains(expected.getDefault()).asAssert()
             } else {
-                plant contains Values(expected.getDefault(), *otherExpected.map { it.getDefault() }.toTypedArray())
+                val values = Values(expected.getDefault(), *otherExpected.map { it.getDefault() }.toTypedArray())
+                plant.asExpect().contains(values(values.expected, *values.otherExpected)).asAssert()
             }
         }
 
         fun getContainsNotDefaultTranslationOfPair()
             = "containsNotDefaultTranslationOf no longer in this API" to Companion::containsNotDefaultTranslationOf
 
 
-        private fun containsNotDefaultTranslationOf(plant: Assert<CharSequence>, expected: Translatable, otherExpected: Array<out Translatable>)
-            = plant containsNot Values(expected.getDefault(), *otherExpected.map { it.getDefault() }.toTypedArray())
+        private fun containsNotDefaultTranslationOf(plant: Assert<CharSequence>, expected: Translatable, otherExpected: Array<out Translatable>): AssertionPlant<CharSequence> {
+            val values = Values(expected.getDefault(), *otherExpected.map { it.getDefault() }.toTypedArray())
+            return plant.asExpect().containsNot(values(values.expected, *values.otherExpected)).asAssert()
```

Comment:
It's not wrong but I would like to see how the ReplaceWith looks like.

Reply:
it's the new ReplaceWith with the Values helper class.

Reply:
really, you have chosen ALT+ENTER `Replace with...` and it introduced a local variable?

Reply:
wow, I just checked it myself, that's quite nice :)